docs/user_guide/mitigation/postprocessing.rst

```diff
@@ -14,69 +14,135 @@ Threshold optimizer
 ^^^^^^^^^^^^^^^^^^^
 
 :class:`ThresholdOptimizer` is based on the paper
-*Equality of Opportunity in Supervised Learning* [#3]_. Unlike other
-mitigation techniques :class:`ThresholdOptimizer` is built to satisfy the
-specified fairness criteria exactly and with no remaining disparity.
+*Equality of Opportunity in Supervised Learning*
+:footcite:`hardt2016equality`.
+Unlike other mitigation techniques :class:`ThresholdOptimizer` is built to
+satisfy the specified fairness criteria exactly and with no remaining
+disparity.
 In many cases this comes at the expense of performance, for example, with
-significantly lower accuracy. Regardless, it is a useful data point to compare
-results with. Importantly, :class:`ThresholdOptimizer` requires the sensitive
-features to be available at deployment time (i.e., for the :code:`predict`
-method).
+significantly lower accuracy.
+Regardless, it is a useful data point to compare results with.

```

Comment:
Nitpick: Sentence ends with a preposition, and I don't think it's making a dramatic effect.

Reply:
rephrased. LMK if that's better